jinahub/indexers/storage/MongoDBStorage/__init__.py

```diff
@@ -80,9 +80,3 @@ def search(self, docs: DocumentArray, parameters: Dict, **kwargs):
 
         with self.handler as mongo_handler:
             mongo_handler.search(docs.traverse_flat(traversal_paths))
-
-    def close(self) -> None:
```

Comment:
why removed?

Reply:
i think the connection close has been properly handdled by the context manager, nothing need to do here @JoanFM